build.gradle

```diff
@@ -89,7 +89,7 @@ dependencies {
 
     annotationProcessor "org.littletonrobotics.akit.junction:junction-autolog:3.0.2"
 
-    implementation 'com.github.galaxia5987:common:2024.1.1'
+    implementation 'com.github.galaxia5987:common:a2c3f5c264'
```

Comment:
Don't commit that here, this should be updated in a separate PR to `develop`, then merged to here (and all other branches).

Reply:
welp, that is to late. but next time I will do so